lib/ui/page/home/page/chat/widget/chat_forward.dart

```diff
@@ -110,6 +111,9 @@ class ChatForwardWidget extends StatefulWidget {
   /// required.
   final FutureOr<RxUser?> Function(UserId userId)? getUser;
 
+  /// Indicator whether this [ChatForwardWidget] should display an [AvatarWidget].
```

Comment:
<img width="642" height="162" alt="Screenshot 2025-09-17 at 12 43 40" src="https://github.com/user-attachments/assets/0cd0a1a2-5290-4c14-b8b5-6343f842e3a9" />

This line exceeds 80 characters per line limit: https://dart.dev/effective-dart/style#prefer-lines-80-characters-or-fewer

You may want to set up your IDE to display the 80th character symbol limit so that it is visible to you when you occasionally exceed that limit.